### DISCUSSION TITLE
Is Tag Replacer still "at your own risk"?

### DISCUSSION THREAD

**Discussion by arcanetrivia** [AprilSylph/XKit-Rewritten#526] [Help Desk] [answered]
I'm happy to see Tag Replacer show up in the features for v.0.16.0, but I wonder: does this extension still come with the caveat that Tumblr may see its activity as somehow suspicious or abusive and result in your blog being suspended/deleted? I stopped using this extension 3 years ago in New XKit because of this issue, but the last I heard about it was back in November 2019 (when it was known to still be happening).

Minor thing: The "help" question mark to jump to its entry in the features list is missing.
![image](https://user-images.githubusercontent.com/68124115/158235233-bcc91387-a7da-4f6d-9554-f5dc2aa8ae6c.png)

**Comment by AprilSylph** [comment, marked as answer]
No way of knowing yet. I assume it's safe since it uses the Mass Post Editor API, which cuts the number of `POST` requests by 50x (for replacing tags) or 100x (for only deleting or appending tags).

The help link will come in the next release. It's missing because I don't like to add information to the wiki before the appropriate release is out.
